quicktests/umd/app.js

```diff
@@ -1,4 +1,5 @@
 /*jshint strict:false */
+/*eslint strict: 0*/
```

Comment:
Note that there is some message about jshint being set to false...

@jtlan Can you chime in on why this should not be in strict mode?